draft-ietf-scitt-architecture.md

```diff
@@ -558,7 +558,8 @@ The subsequent sections describe the main concepts, namely Transparency Service,
 ## Transparency Service
 
 Transparency Services MUST feature an Append-only Log.
-The Append-only Log is the verifiable data structure that records registered Signed Statements and supports the production of Receipts.
+The Append-only Log is the verifiable data structure that records integrity protection of the registered Signed Statements and supports the production of Receipts.
+Transparency Services may provide ancillary services to store the contents of the Signed Statement, however the Append-only Log does not store the Signed Statement contents avoiding concerns of persisting personally identifiable information.
```

Comment:
```suggestion
Transparency Services may provide ancillary services to store the contents of the Signed Statement. The Append-only Log MUST store a one-way commitment binding the data to the log. The commitment MAY also be hiding.
```
A sha256 hash, for example, satisfies the MUST, but is not hiding (proving the statement is on the log reveals the statement to the prover). A pedersen commitment would be both binding and hiding. A commitment is probably a general enough concept that this does not limit the possible implementations ?

Reply:
This architecture is limited to "signed statements", the payload can be whatever you want it to be, but the signed statement is a cose sign1.

I don't think normative language belongs in this document beyond:

A signed statement is a cose-sign1, ... the verification procedure is define in ....

and

A receipt is a cose-sign1, ... the verification precedure is defined in ...

These are normative statements, they just don't use MUST.

I don't think we can get consensus to say "payload of a signed statement MUST/MAY/SHOULD ...."